src/Uno.UI/ts/Windows/Devices/Sensors/LightSensor.ts

```diff
@@ -0,0 +1,51 @@
+﻿declare class AmbientLightSensor {
```

Comment:
I wonder. Should we place this in another file ? @carldebilly

Reply:
Was is the difference between a _light sensor_ and an _ambiant light sensor_?

Reply:
@carldebilly `LightSensor` is the API name in UWP, I try to match the .ts file names with the same name as in C#, so it is easy to find. However, in the browser the API is `AmbientLightSensor`. Because the API is not yet "stable", I had to add this temporary declaration to be able to work with it in Typescript. I have used similar approach for other APIs like Accelerometer